### DISCUSSION TITLE
Support more OS and arch?

### DISCUSSION THREAD

**Discussion by kappa-maintainer** [burningwave/jvm-driver#13] [Ideas]
I want to employ burningwave core in my project using LWJGL3. LWJGL3 itself supports more platforms than jvm driver, I wonder if jvm driver has plan to support more OS and Arch too?
[Here](https://github.com/LWJGL-CI/lwjgl3/blob/91ed5ea015ab6a1c24f4d104fa2653427cb29aae/build.gradle.kts#L61)'s the platforms LWJGL supported, and [here](https://github.com/LWJGL-CI/lwjgl3/blob/master/.github/workflows/build.yml)'s how their action handle them.

**Comment by kappa-maintainer** [comment]
Actually, our [project](https://github.com/CleanroomMC/Cleanroom) could use burning wave in many places, and we have reinvented the [wheel](https://github.com/Rongmario/ImagineBreaker) earlier before I found this project.

**Comment by Roberto-Gentili** [comment]
> I want to employ burningwave core in my project using LWJGL3. LWJGL3 itself supports more platforms than jvm driver, I wonder if jvm driver has plan to support more OS and Arch too? [Here](https://github.com/LWJGL-CI/lwjgl3/blob/91ed5ea015ab6a1c24f4d104fa2653427cb29aae/build.gradle.kts#L61)'s the platforms LWJGL supported, and [here](https://github.com/LWJGL-CI/lwjgl3/blob/master/.github/workflows/build.yml)'s how their action handle them.

If you use the default driver you should have no problems on any platform except Android. The concept of supported platforms applies only to drivers that use or could use the JNI interface. I'm currently not able to compile the JNI module for platforms other than those indicated in the [documentation](https://github.com/burningwave/jvm-driver?tab=readme-ov-file#overview), but if you can [you're welcome](https://github.com/burningwave/jvm-driver?tab=readme-ov-file#contributing) and you can create a pull request

**Comment by kappa-maintainer** [comment]
We met a guy that help us to compile our JNI `openAllModulesToAll` approach (switched to method/varhandles later, doubt it will still work in later java though) to Android, will check about that
